### ISSUE TITLE
learning rate setting

### ISSUE THREAD

**Issue by suoluowan** [msracver/Flow-Guided-Feature-Aggregation#21]
In your paper you set lr = 0.001 when train on the first 80k in ImageNet DET training set, and set lr = 0.0001 when train on the last 40k in ImageNet DET training set, and so on. so I want to know if the learning rate will change when training the FGFA end-to-end?
